### PR TITLE
[ISSUE-277] Create Coronavirus Information Page

### LINKED ISSUE #277: Add "COVID-19 Information" Page
COVID-19 measures are going to be around for a good while, even when the space is reopened. 

Let's put up a single information page for members and keep it up to date.

### PR DESCRIPTION
**Pull request by awflwafl**
Closes #277.

## Description

Create a Coronavirus information page for members and visitors that will be kept up to date and reviewed periodically.

Additionally...

* Update header link
* Update link visitor pass purchase
* Update link under membership purchase

New Page |
--- |
image |

Before | After
--- | ---
image | image
image | image
image | image

### CODE REVIEW

File: coronavirus.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+* **❌ Visitors: No**
+* **❌ Members: No**
+* **💻 Events and meetings: Remote**
+
+_Last updated on the 19th of May 2020._
+
+---
+
+### Asked Questions
```
Comment (oibind) [inline coronavirus.md:17]: FAQs

Comment (awflwafl) [reply coronavirus.md:17]: Can you add some reasoning, please?

File: _layouts/default.html [hunk 2]
```diff
@@ -156,7 +156,7 @@ <h1 class="banner"><a href="/"><span class="fsl-purple">Farset</span><span class
         <div id="covid-19-statement" class="row">
           <div class="small-12 columns" style="height:45px; display: flex; align-items: center;">
             <div style="display: flex; align-items: center; width: 100%; height: 100%; padding: 0 15px; background-color: #ffd411;">
-              <a href="https://blog.farsetlabs.org.uk/2020/03/covid-19-building-temporarily-closing/" style="font-size: 0.9em; font-weight: bold;">Building Closing on 27th of March (COVID-19)</a>
+              <a href="/coronavirus" style="font-size: 0.9em; font-weight: bold;">Coronavirus Information for Members and Visitors</a>
```
Comment (oibind) [inline _layouts/default.html:159]: COVID19: Information for Members and Visitors.

File: coronavirus.md (hunk 1 above)
Comment (oibind) [reply coronavirus.md:17]: Not that I'm personally prescriptive in anyway but it's an already well established phrase/initialism.

Comment (awflwafl) [reply coronavirus.md:17]: Ah, makes sense. I'll update the heading.

**Comment by oibind** [comment]
I'll approve once the changes have been pushed.

**Pull request merged**